contribs/application/src/main/java/org/matsim/application/analysis/noise/NoiseAnalysis.java

```diff
@@ -0,0 +1,134 @@
+package org.matsim.application.analysis.noise;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.matsim.api.core.v01.Coord;
+import org.matsim.api.core.v01.Scenario;
+import org.matsim.application.MATSimAppCommand;
+import org.matsim.application.options.CrsOptions;
+import org.matsim.application.options.ShpOptions;
+import org.matsim.contrib.noise.MergeNoiseCSVFile;
+import org.matsim.contrib.noise.NoiseConfigGroup;
+import org.matsim.contrib.noise.NoiseOfflineCalculation;
+import org.matsim.contrib.noise.ProcessNoiseImmissions;
+import org.matsim.core.config.Config;
+import org.matsim.core.config.ConfigUtils;
+import org.matsim.core.scenario.ScenarioUtils;
+import org.matsim.core.utils.geometry.CoordinateTransformation;
+import picocli.CommandLine;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@CommandLine.Command(
+        name = "noise-analysis",
+        description = "Noise analysis",
+        mixinStandardHelpOptions = true,
+        showDefaultValues = true
+)
+
+public class NoiseAnalysis implements MATSimAppCommand {
+    private static final Logger log = LogManager.getLogger(NoiseAnalysis.class);
+
+    @CommandLine.Option(names = "--directory", description = "Path to run directory", required = true)
+    private String runDirectory;
+
+    @CommandLine.Option(names = "--runId", description = "Pattern to match runId.", defaultValue = "")
+    private String runId;
+
+    @CommandLine.Mixin
+    private CrsOptions crs = new CrsOptions();
+
+    @CommandLine.Mixin
+    private ShpOptions shp = new ShpOptions();
+
+
+    public static void main(String[] args) {
+        System.exit(new CommandLine(new NoiseAnalysis()).execute(args));
+    }
+
+    @Override
+    public Integer call() throws Exception {
+        Config config = ConfigUtils.createConfig(new NoiseConfigGroup());
+        config.global().setCoordinateSystem(crs.getInputCRS());
+        config.controler().setRunId(runId);
+        if (runId != "") {
+            config.network().setInputFile(runDirectory + "/" + runId + ".output_network.xml.gz");
+            config.plans().setInputFile(runDirectory + "/" + runId + ".output_plans.xml.gz");
+        } else {
+            config.network().setInputFile(runDirectory + "/" + runId + "output_network.xml.gz");
+            config.plans().setInputFile(runDirectory + "/" + runId + "output_plans.xml.gz");
+        }
+        config.controler().setOutputDirectory(runDirectory);
+
+        // adjust the default noise parameters
+        NoiseConfigGroup noiseParameters = ConfigUtils.addOrGetModule(config, NoiseConfigGroup.class);
+        noiseParameters.setReceiverPointGap(250);
```

Comment:
The receiver point gap might be an option configurable as command line option. There should be very few magic numbers in the command scripts.